server/controllers/index.ts

```diff
@@ -46,4 +51,30 @@ export abstract class BaseController extends FormWizard.Controller {
     }
     return error
   }
+
+  csrfGenerateSecret(req: FormWizard.Request, res: Express.Response, next: Express.NextFunction): void {
+    // copy application middleware CSRF token into form wizard for sanity
+    req.sessionModel.set('csrf-secret', res.locals.csrfToken)
+    next()
+  }
 }
+
+Object.assign(FormWizard.Controller.validators, {
```

Comment:
unfortunately, the validators set is not read from an _instance_ of a controller so we have to add to the global set